database/migrations/2016_02_09_224824_create_competitions_table.php

```diff
@@ -14,9 +14,10 @@ public function up()
     {
         Schema::create('competitions', function (Blueprint $table) {
             $table->increments('id');
-            $table->date('start_date');
-            $table->date('end_date');
+            $table->integer('campaign_id')->unsigned()->index();
             $table->integer('campaign_run_id')->unsigned()->index();
+            $table->date('start_at');
```

Comment:
I'm pretty sure the standard is for date fields to explicitly say the word date in them
where `at` is more about when content is created/edited/removed